x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -53,6 +55,31 @@ export function registerJobsHealthAlertingRule(
         );
       }
 
+      if (
+        !!resultTestConfig.delayedData.timeInterval &&
+        validateLookbackInterval(resultTestConfig.delayedData.timeInterval)
```

Comment:
There is a case when the user can assign all jobs (with `*`). It means the alerting executor will pick up jobs created _after_  the alerting rule instance is created. So when the `timeInterval` is set to `null` (basically empty), the server-side logic will derive the time interval from the longest bucket length. I think we should also multiple it by 2 just in case. What do you think?

Reply:
OK, a default of at least 2 bucket lengths makes sense to me. Consequently though, that default should be set before constructing the query, which I didn't see occur.

Reply:
Added in https://github.com/elastic/kibana/pull/107183/commits/eb66a6830582799d14789e5735942299c8631f1e and https://github.com/elastic/kibana/pull/107183/commits/a4e0bc5576257e02d982fc25e308d0df08d29cd7